install/kubernetes/helm/Istio/requirements.yaml

```diff
@@ -0,0 +1,13 @@
+dependencies:
+  - name: pilot
+    version: 0.1.0-1
+    condition: Istio.pilot_enabled
```

Comment:
pilot will always be enabled - hence not require a condition variable.

Reply:
I disagree, in some cases deploying just generic configuration might be useful.

Reply:
So pilot is a required component. Without that there is nothing to configure Envoy fleet or setup routing in sidecars.

Reply:
@sbezverk istio is non-functional without pilot as it is responsible for managing the data-plane.

Reply:
Gents, I am also thinking about development flow. I mean I change config redeploy it and I want to check if everything is ok. The pilot switch can be defaulted to enabled for 99.99% cases, but when you need a stop before further deploying, I think it is still useful to have, it will not take too much space ;)